lib/lifecycles/changelog.js

```diff
@@ -52,7 +51,7 @@ function outputChangelog (args, newVersion) {
 
 function createIfMissing (args) {
   try {
-    accessSync(args.infile, fs.F_OK)
+    fs.accessSync(args.infile)
```

Comment:
`F_OK` is the default so removed that since `fs.F_OK` is also deprecated in favor of `fs.constants.F_OK`